Automattic/wp-calypso
# Detect skipped checkout in launchpad selected plan

Whenever a user skips purchasing the plan he selected we should customize the launchpad experience.
1. The `Choose a plan (plan_selected)` task should be marked an uncompleted
2. The header should change from "Let's get ready to launch" to "Get started with WordPress.com"

Fixes https://github.com/Automattic/wp-calypso/issues/92243

## Proposed Changes

* Redirect with param when closing checkout
* Apply the launchpad customizations based on this 


## Testing Instructions
- Follow the onboarding flow `/start`
- Select a paid plan
- Click the close checkout button
- Ensure that `skippedCheckout` is added to the URL
- Don't select goals so that we default with the build intent
- Select a design
- Ensure that the customizations for the title and selected plan are applied to the launchpad

### Testing flows from stepper
-  Follow the `/start/newsletter` flow
- Select a paid plan
- Click the close checkout button
- Ensure that `skippedCheckout` is added to the URL
- Ensure that the selected plan task is marked as incomplete



*

## Pre-merge Checklist
